rootfs/etc/nginx/lua/stratio_auth.lua

```diff
@@ -45,7 +45,10 @@ local function create_jwt(oauth2_cookie, userinfo_url, signing_key)
                 groups = userinfo["groups"],
                 mail = userinfo["email"],
                 tenant = userinfo["tenant"],
-                uid = userinfo["user"]
+                uid = userinfo["user"],
+                username = userinfo["username"],
+                tenants = userinfo["tenants"],
+                equipo = "gosec"
```

Comment:
sólo porque lo he visto cuando nos comentábais esto. 
¿No sería mejor si todas las variables son en inglés mantenenerlo así?.
```suggestion
                team = "gosec"
```
Que igual al ser prueba esto no va luego a ningún sitio, pero bueno, por si acaso.

Reply:
si perdona, es una prueba para una poc, de hecho ese campo ni va a ir, solo era para testear